src/Euclidean.jl

```diff
@@ -183,3 +185,8 @@ function normal_tvector_distribution(M::Euclidean{Tuple{N}}, x, σ) where N
     d = Distributions.MvNormal(zero(x), σ)
     return ProjectedFVectorDistribution(TangentBundleFibers(M), x, d, project_vector!, x)
 end
+
+mean(M::Euclidean, x::AbstractVector, w::AbstractWeights) = mean(x,w)
+mean(M::Euclidean, x::AbstractVector) = mean(x)
```

Comment:
Should also have `var` here. And probably also `mean!`.

Reply:
and median!

Reply:
And I thought this would be one of the shortest PRs I''ll do, it's getting the longest, actually – but yes you're right.

Reply:
Sorry!

Reply:
The problem with `median!` is, that in `Statistics.jl` there is no two argument `median!` which I find very strange.

Reply:
Oh that is surprising. StatsBase doesn't have it either. Okay, then we should probably just have an overload that computes `median` and then copies it.

Reply:
I don'T follow you here. What do you mean? Can you propose a line of code?

Reply:
```julia
median!(M::Euclidean, y, x::AbstractVector, w::AbstractWeights) = copyto!(y, median(x, w))
median!(M::Euclidean, y, x::AbstractVector) = copyto!(y, median(x))
```

Reply:
Thanks, I'll add that then.